services/venice-controller/src/main/java/com/linkedin/venice/controller/Admin.java

```diff
@@ -289,26 +288,9 @@ Version incrementVersionIdempotent(
       String targetedRegions,
       int repushSourceVersion);
 
-  String getRealTimeTopic(String clusterName, Store store);
+  Version getIncrementalPushVersion(String clusterName, String storeName, String pushJobId);
 
-  default String getRealTimeTopic(String clusterName, String storeName) {
-    Store store = getStore(clusterName, storeName);
-    if (store == null) {
-      throw new VeniceNoStoreException(storeName, clusterName);
-    }
-    return getRealTimeTopic(clusterName, store);
-  }
-
-  String getSeparateRealTimeTopic(String clusterName, String storeName);
```

Comment:
It seems like these two APIs are removed. I am a bit lost here, do we move the logic to a certain method that ensures RT is created? Could you point me to that part of the code?

Reply:
We moved RT topic creation logic in `addVersion`. This way we won't have to worry about mismatch between partition count in version topic and RT topic. This also helps us to simplify the logic for RT repartition project 

Code is in `VHA:: createOrUpdateRealTimeTopics`

Reply:
Thanks for offline explanation, got it now.